### ISSUE TITLE
FTBFS: error: variable ‘QDataStream ds’ has initializer but incomplete type

### ISSUE THREAD

**Issue by onlyjob** [QuiteRSS/quiterss#858]
As reported in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=805240
QuiteRSS 0.18.2 FTBFS as follows:

```
3rdparty/qtsingleapplication/qtlocalpeer.cpp: In member function ‘bool QtLocalPeer::sendMessage(const QString&, int)’:
3rdparty/qtsingleapplication/qtlocalpeer.cpp:160:19: error: variable ‘QDataStream ds’ has initializer but incomplete type
     QDataStream ds(&socket);
                   ^
3rdparty/qtsingleapplication/qtlocalpeer.cpp: In member function ‘void QtLocalPeer::receiveConnection()’:
3rdparty/qtsingleapplication/qtlocalpeer.cpp:180:26: error: variable ‘QDataStream ds’ has initializer but incomplete type
     QDataStream ds(socket);
                          ^
Makefile.Release:2624: recipe for target 'release/obj/qtlocalpeer.o' failed
make[2]: *** [release/obj/qtlocalpeer.o] Error 1
make[2]: Leaving directory '/quiterss-0.18.2+dfsg'
Makefile:38: recipe for target 'release' failed
```

Full build log:
https://reproducible.debian.net/rb-pkg/unstable/amd64/quiterss.html

**Comment by onlyjob** [comment]
A month passed... Any news about this problem?

**Comment by onlyjob** [comment]
Fixed by 798c831b. Looking forward new release...

**Comment by onlyjob** [comment]
Fixed in 0.18.3, thanks.

**Comment by Funcy-dcm** [comment]
:)
